main.cpp

```diff
@@ -74,23 +74,29 @@ int main()
 
     std::cout << "Number of channels: " << channelWrapper.pointerToChannelStrip->addChannel(4) << " Number of inputs: " << channelWrapper.pointerToChannelStrip->numInputs << std::endl;
 
-    channelWrapper.pointerToChannelStrip->func();
+    std::cout << "Number of channels: " << channelWrapper.pointerToChannelStrip->addChannel(4) << " Number of inputs: " << channelWrapper.pointerToChannelStrip->numInputs << std::endl;
+
+    // channelWrapper.pointerToChannelStrip->func();
 
     //=========================================================
 
     EqualizerWrapper eqWrapper (new Equalizer);
 
     std::cout << "Number of bands: " << eqWrapper.pointerToEqualizer->addBand() << " Max Freq Range: " << eqWrapper.pointerToEqualizer->maxFreqRange << std::endl;
 
-    eqWrapper.pointerToEqualizer->func();
+    std::cout << "Number of bands: " << eqWrapper.pointerToEqualizer->addBand() << " Max Freq Range: " << eqWrapper.pointerToEqualizer->maxFreqRange << std::endl;
+
+    // eqWrapper.pointerToEqualizer->func();
 
     //=========================================================
 
     PreampWrapper preWrapper (new Preamp);
 
     std::cout << "How much mid reduction: " << preWrapper.pointerToPreamp->cutMud(12.0f) << " Gain increase: " << preWrapper.pointerToPreamp->gainIncrease << std::endl;
 
-    preWrapper.pointerToPreamp->func();
+    std::cout << "How much mid reduction: " << preWrapper.pointerToPreamp->cutMud(12.0f) << " Gain increase: " <<  preWrapper.pointerToPreamp->gainIncrease << std::endl;
+
+    // preWrapper.pointerToPreamp->func();
```

Comment:
You need to continue to call `func()` the way you did before.
Move `func()` to the cpp file.
- [ ] revise